### ISSUE TITLE
ldap接入问题

### ISSUE THREAD

**Issue by weiliang-ms** [kubesphere/ks-installer#532]
参考以下文章接入公司内部Ldap
[ldap接入](https://kubesphere.com.cn/docs/v2.1/zh-CN/installation/ldap-ad/#___gatsby)

![image](https://user-images.githubusercontent.com/18642754/71640367-787e2800-2cc3-11ea-8c90-6191888aaa0b.png)
查看pod日志
![image](https://user-images.githubusercontent.com/18642754/71640370-8764da80-2cc3-11ea-852f-34ffc07c6eee.png)
redis状态
![image](https://user-images.githubusercontent.com/18642754/71640374-ac594d80-2cc3-11ea-8dfd-38df3f7f3f04.png)
redis日志
![image](https://user-images.githubusercontent.com/18642754/71640379-c135e100-2cc3-11ea-9fe5-1844e110d756.png)
ldap状态
![image](https://user-images.githubusercontent.com/18642754/71640420-8a13ff80-2cc4-11ea-9e05-8831d741be95.png)

ldap日志
![image](https://user-images.githubusercontent.com/18642754/71640422-9d26cf80-2cc4-11ea-85ca-a0ead1be5ff8.png)

**Comment by weiliang-ms** [comment]
请问有什么排查的方向吗？

**Comment by wansir** [comment]
@weiliang-ms  
`kubectl -n kubesphere-system logs ks-account-xxxxx -c ad-sidecar` 检查一下日志

**Comment by weiliang-ms** [comment]
感谢，ldap配置的问题，现在已经开始同步了

**Comment by halinwu** [comment]
I followed the AD setting and change some parameters , but when I excuted the inject-ks-account.sh , I got some error message ( Error from server (NotFound): deployments.apps "ks-account" not found ) . I have some questions, pls guide me to fix the issue. 
1. how to set  $host , AD Server or control node ?    # 将 host的值改为实际的服务器 IP 与 端口 
2. how to distinguish the src ManagerDN and dst ManagerDN,  all of them must match AD DN ?  
3. if we set those parameter at the inject-ks-account.sh , any ohter would be change at  ks-installer （kubectl edit cc ks-installer） ？
